lighthouse-extension/app/src/popup.js

```diff
@@ -128,6 +128,10 @@ document.addEventListener('DOMContentLoaded', _ => {
     return frag;
   }
 
+  if (background.isRunning()) {
+    startSpinner();
```

Comment:
Yes whenever runner fails promise then or catch than we do a stopSpinner. Not sure if Start and Stop spinner is still a good name for it. Maybe something a long the lines of showProcessPage or just showProcess?